Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt

```diff
@@ -0,0 +1,114 @@
+class ProfileListFragmentTest : BaseUITest() {
+
+    @MockK lateinit var viewModel: ProfileListViewModel
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this, relaxed = true)
+
+        every { viewModel.profiles } returns profileCards()
+
+        setupMockViewModel(
+            object : ProfileListViewModel.Factory {
+                override fun create(): ProfileListViewModel = viewModel
+            }
+        )
+    }
+
+    @After
+    fun teardown() {
+        clearAllViewModels()
+    }
+
+    @Test
+    @Screenshot
+    fun capture_fragment() {
+        launchFragmentInContainer2<ProfileListFragment>()
+        takeScreenshot<ProfileListFragment>("1")
+    }
+
+    private fun profileCards(): LiveData<List<ProfileListItem>> =
+        MutableLiveData(
+            listOf<ProfileListItem>(
+                ProfileCard.Item(
+                    profile = Profile(
+                        id = 1,
+                        firstName = "Andrea",
+                        lastName = "Schneider",
+                        birthDate = LocalDate(1990, 11, 17),
+                        city = "Potsdam",
+                        zipCode = "14471",
+                        street = "Lange Straße 5",
+                        phone = "0151123456789",
+                        email = "andreaschneider@web.de"
+                    ),
+                    qrCode = "BEGIN:VCARD\n" +
+                        "VERSION:4.0\n" +
+                        "N:Schneider;Andrea;;;\n" +
+                        "FN:Andrea Schneider\n" +
+                        "BDAY:19690420\n" +
+                        "EMAIL;TYPE=home:andrea.schneider@gmail.com\n" +
+                        "TEL;TYPE=\"cell,home\":(359) 260-345\n" +
+                        "ADR;TYPE=home:;;Long Str\\, No. 4;Liverpool;;12345\n" +
+                        "REV:20220418T091325Z\n" +
+                        "END:VCARD",
+                    onClickAction = { _, _ -> }
+                ),
+
+                ProfileCard.Item(
+                    profile = Profile(
+                        id = 2,
+                        firstName = "Bax",
+                        lastName = "Bustermann",
+                        birthDate = LocalDate(1997, 7, 24),
+                        city = "Potsdam",
+                        zipCode = "14471",
+                        street = "Shorte Straße 5",
+                        phone = "0234523454545",
+                        email = "baxbustermann@web.de"
+                    ),
+                    qrCode = "BEGIN:VCARD\n" +
+                        "VERSION:4.0\n" +
+                        "N:Busterman;Bax;;;\n" +
+                        "FN:Bax Busterman\n" +
+                        "BDAY:19690420\n" +
+                        "EMAIL;TYPE=home:max.musterman@gmail.com\n" +
+                        "TEL;TYPE=\"cell,home\":(359) 260-345\n" +
+                        "ADR;TYPE=home:;;Long Str\\, No. 4;Liverpool;;12345\n" +
```

Comment:
Does the qr code data need to match? I thought they are picky about screen shots as they provide them to the public

Reply:
No, I don't think so, I just tried to keep it consistent.